releasenotes/notes/add-aux-unit-support-curve-analysis-beb4150ef5de283b.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    Parameter reported by :py:class`qiskit_experiments.curve_analysis.curve_analysis.CurveAnalysis`
+    subclass can be scaled with auxiliary unit if unit is provided. For example,
+    Frequency = 1000 ± 0.1 Hz is now displayed as 1 ± 0.0001 kHz.
```

Comment:
If `unit` is provided and `scale` is `True`. By default `scale` is False, which is the desired behavior. In fact, where do you allow the user to control `scale`?

Reply:
Now scale is removed from fitval method.